WordPress/theme-experiments
# Rename theme to TT1 Blocks

This PR addresses the name change part of #140.

Based on the discussion in the recent [block-based themes meeting](https://make.wordpress.org/themes/2020/12/16/block-themes-meeting-notes-for-16-december-2020/), folks seemed okay with the name "TT1 Blocks". 

Note that we will need to follow up with a Gutenberg PR to address some tests there: https://github.com/WordPress/gutenberg/search?q=twentytwentyone-blocks

**To test**
- Clone this repository into a fresh site's themes directory
- Install and activate Gutenberg 
- Ensure the site and site editor load as expected

One question — could we update the theme screenshot to somehow visually represent that this is not the same theme as Twenty Twenty-One?
